resolwe/flow/management/commands/collecttools.py

```diff
@@ -0,0 +1,133 @@
+""".. Ignore pydocstyle D400.
+
+========================
+Collect Processes' tools
+========================
+
+"""
+import os
+from pathlib import Path
+import shutil
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+from resolwe.flow.utils import get_apps_tools
+
+
+class Command(BaseCommand):
+    """Copies tools files from different locations to the settings.TOOLS_ROOT."""
+
+    help = "Collect tools files in a single location."
+
+    def __init__(self, *args, **kwargs):
+        """Initialize command."""
+        super().__init__(*args, **kwargs)
+        self.interactive = None
+        self.clear = None
+
+        if not hasattr(settings, 'FLOW_TOOLS_ROOT'):
+            raise CommandError("FLOW_TOOLS_ROOT must be defined in Django settings.")
+
+        self.destination_path = settings.FLOW_TOOLS_ROOT
+
+    def add_arguments(self, parser):
+        """Command arguments."""
+        parser.add_argument(
+            '--noinput', '--no-input', action='store_false', dest='interactive',
+            help="Do NOT prompt the user for input of any kind.",
+        )
+
+        parser.add_argument(
+            '-c', '--clear', action='store_true', dest='clear',
+            help="Clear the existing files using the storage before trying to copy or link the "
```

Comment:
"Clear the existing files using the storage" is not a clear description. Can you improve it?